lib/core.js

```diff
@@ -201,14 +229,14 @@ var _XDomainRequest = window.XDomainRequest;
  */
 Core.prototype.reportError = function(e) {
   var that = this;
-  if (this._id) {
-    return this.id().then(report);
+  if (this._ids) {
+    return this._ids.then(report, function() { report(); });
```

Comment:
This is a "catch" right? You should be able to call the same way as the first parameter.

```suggestion
    return this._ids.then(report, report);
```

Reply:
The `function() { report(); }` should really be a `function(err) { report(); }`. The wrapper is to discard the error value so it's not passed as the argument to `report` which is already reporting an earlier error.

I suppose I could do:

```js
return this._ids.then(undefined, function() { /* swallow error */ }).then(report);
```

Reply:
Changed in https://github.com/unravelin/ravelinjs/pull/54/commits/cb3ae12e85b59cd463e1d1e41bd63365c0511d43.